packages/apputils/src/sessioncontext.tsx

```diff
@@ -1410,14 +1416,20 @@ export class SessionContextDialogs implements ISessionContext.IDialogs {
       throw new Error('No kernel to restart');
     }
 
-    const restartBtn = Dialog.warnButton({ label: 'Restart' });
+    const restartBtn = Dialog.warnButton({
+      label: 'Restart',
```

Comment:
It seems it was missing previously, let's add it in:

```suggestion
      label: trans.__('Restart'),
```